packages/react-native-reanimated/src/layoutReanimation/defaultTransitions/FadingTransition.ts

```diff
@@ -29,7 +29,7 @@ export class FadingTransition
     const delayFunction = this.getDelayFunction();
     const callback = this.callbackV;
     const delay = this.getDelay();
-    const duration = this.durationV ?? 500;
+    const duration = (this.durationV ?? 500) / 2;
```

Comment:
```suggestion
    const duration = this.durationV / 2 ?? 250;
```

Reply:
This is great change! But this change is beyond the scope of this PR. Could you please move it to a separate PR, and then rebase this PR onto the new one with the updated defaults?

Reply:
Yes, I just want to merge the other PR first